cl/search/api_utils.py

```diff
@@ -57,19 +54,13 @@ def get_object_list(request, cd, paginator, es_flag_status):
     if cd["type"] == SEARCH_TYPES.DOCKETS:
         group = True
 
-    is_oral_argument_active = (
-        cd["type"] == SEARCH_TYPES.ORAL_ARGUMENT and es_flag_status
-    )
-    is_people_active = cd["type"] == SEARCH_TYPES.PEOPLE and es_flag_status
-    is_opinion_active = cd["type"] == SEARCH_TYPES.OPINION and es_flag_status
-    is_recap_active = (
-        cd["type"]
-        in [
-            SEARCH_TYPES.RECAP,
-            SEARCH_TYPES.DOCKETS,
-        ]
-        and es_flag_status
-    )
+    is_oral_argument_active = cd["type"] == SEARCH_TYPES.ORAL_ARGUMENT
+    is_people_active = cd["type"] == SEARCH_TYPES.PEOPLE
+    is_opinion_active = cd["type"] == SEARCH_TYPES.OPINION
+    is_recap_active = cd["type"] in [
+        SEARCH_TYPES.RECAP,
+        SEARCH_TYPES.DOCKETS,
+    ]
```

Comment:
A `match-case` structure could simplify this block and the following if statements.

Reply:
Yeah! I've applied the suggested refactor